conda/recipes/librmm/meta.yaml

```diff
@@ -85,25 +79,15 @@ outputs:
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
         - {{ compiler('cuda') }}
-        {% if cuda_major != "11" %}
-        - cuda-cudart-dev
-        {% endif %}
+        - {{ "cuda-cudart-dev" if not (cuda_major == "11") else "cuda-version" }}
```

Comment:
Similar, we don't need an "else" here.

Reply:
all of these changes were to try to make `conda-recipe-manager convert` happy -- I'm doing to delete `meta.yaml` entirely as conversion is currently more trouble than it's worth

Reply:
Yep let's go ahead and do that